script.js

```diff
@@ -22,3 +22,25 @@ function hideMenuFunction() {
 menuBar.addEventListener('click', showMenuFunction);
 cancelIcon.addEventListener('click', hideMenuFunction);
 sideItems.addEventListener('click', () => { sideNav.style.display = 'none'; });
+
+const form = document.querySelector('form');
+function handleSubmit(event) {
+  event.preventDefault();
+
+  const emailInput = document.querySelector('#email');
+  const email = emailInput.value;
+
+  if (email === email.toLowerCase()) {
+    form.submit();
+  } else {
+    let errorMessage = form.querySelector('.errorMsg');
+    if (errorMessage) {
+      form.removeChild(errorMessage);
+    }
+    errorMessage = document.createElement('p');
+    errorMessage.classList.add('errorMsg');
+    errorMessage.textContent = 'Oops! An error has occurred, your email must be lowercase.';
```

Comment:
Well done on implementing email validation in your project! 👏🎉 However, it appears that the visual presentation of the validation feedback is not consistent across different devices, as shown in the video. To ensure a better user experience and visual consistency, it's important to address this issue.

[Edgah-Portfolio.webm](https://github.com/Edgahkipkemoi/mobile/assets/55840999/ba13a5c1-48f6-446b-aeff-c8834ac72433)

Consider updating your code to provide visually appealing and consistent validation feedback that works well on various devices. This could involve adjusting the styling, such as color, font, or placement, of the validation message to ensure it's easily noticeable and readable for users.

Taking the time to enhance the visual presentation of the email validation feedback will contribute to the overall professionalism and usability of your project. Keep up the great work and continue refining your implementation! 🌟👍